cli/flox-rust-sdk/src/models/manifest/raw.rs

```diff
@@ -238,6 +238,7 @@ impl RawManifest {
             } => {
                 profile_table.decor_mut().set_suffix(indoc! {r#"
 
+                    ## Set your aliases and stuff here, too.
```

Comment:
Non-blocking: 

For wording and style I'll defer to @jennymahmoudi @garbas; personally I think this might rather belong into the "header" part above.

Reply:
I went ahead and made this change so we can get this merged